Documentation/DocuBlocks/foxxStartup.md

```diff
@@ -0,0 +1,34 @@
+
+@startDocuBlock foxxStartup
+@brief enable or disable the Foxx service propagation on startup
+`--foxx.force-update-on-startup flag`
+
+if *true*, all Foxx services in all databases will be synchronized between
+multiple coordinators during the boot sequence. This ensures that all Foxx
+services are up-to-date when a coordinator reports itself as ready.
+
+In case the option is set to `false` (i.e. no waiting), the coordinator 
+will complete the boot sequence faster, and the Foxx services will be 
+propagated lazily. Until the initialization procedure has completed for
+the local Foxx apps, any request to a Foxx app will be responded to with
+an HTTP 500 error and message 
```

Comment:
```suggestion
an HTTP 503 error and message 
```

Reply:
It actually returns HTTP 500, even though in my code I used 503. This is due to some exception rewriting inside Foxx.